webview/src/setup/components/dvc/CliUnavailable.tsx

```diff
@@ -58,6 +78,35 @@ export const CliUnavailable: React.FC<PropsWithChildren> = ({ children }) => {
       <h1>DVC is currently unavailable</h1>
       {children}
       {conditionalContents}
+      <div className={styles.sideBySideButtons}>
+        <span className={styles.buttonWrapper}>
```

Comment:
Ran into Tippy warnings that lead me to adding extra span wrappers on the buttons.